src/components/specific/models/models-manager/generic-models-manager/GenericModelsManager.vue

```diff
@@ -150,21 +177,31 @@ export default {
       await download(models);
     };
 
+    const updateUpload = async file => {
+      if (!file?.model_id) await createModel(props.project, file);
```

Comment:
If an inline if with a return is OK, I really don't like inline if with real stuff. Can you use brackets?